libcloud/dns/drivers/auroradns.py

```diff
@@ -239,18 +572,34 @@ def __res_to_record(self, zone, record):
         extra['ttl'] = record['ttl']
         extra['prio'] = record['prio']
 
-        return Record(id=record['id'], name=name, type=record['type'],
-                      data=record['content'], zone=zone, driver=self,
-                      ttl=record['ttl'], extra=extra)
+        return Record(id=record['id'], name=name,
+                      type=record['type'],
+                      data=record['content'], zone=zone,
+                      driver=self, ttl=record['ttl'],
+                      extra=extra)
 
     def __res_to_zone(self, zone):
-        return Zone(id=zone['id'], domain=zone['name'], type=DEFAULT_ZONE_TYPE,
+        return Zone(id=zone['id'], domain=zone['name'],
+                    type=DEFAULT_ZONE_TYPE,
                     ttl=DEFAULT_ZONE_TTL, driver=self,
                     extra={'created': zone['created'],
                            'servers': zone['servers'],
                            'account_id': zone['account_id'],
                            'cluster_id': zone['cluster_id']})
 
+    def __res_to_healthcheck(self, zone, healthcheck):
+        return AuroraDNSHealthCheck(id=healthcheck['id'],
+                                    type=healthcheck['type'],
+                                    hostname=healthcheck['hostname'],
+                                    ipaddress=healthcheck['ipaddress'],
+                                    health=healthcheck['health'],
+                                    threshold=healthcheck['threshold'],
+                                    path=healthcheck['path'],
+                                    interval=healthcheck['interval'],
+                                    port=healthcheck['port'],
+                                    enabled=healthcheck['enabled'],
+                                    zone=zone, driver=self)
```

Comment:
I don't know how this method is used. but you should use self.connection.driver. If there are multiple drivers instantiated weird things would happen with this reference!